### PR TITLE
feat: adds accordion config to component explorer

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Adds `fast-accordion` config a more complex config file for the component explorer

Screen Shot 2020-06-10 at 4 08 48 PM

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [x] I have updated the [definition file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [x] I have updated the [configuration file](https://github.com/Microsoft/fast-dna/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/site-utilities/src/definitions/fast-components/fast-accordion.definition.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                        },
+                        {
+                            name: "multi",
+                        },
+                    ],
+                    default: "multi",
+                    required: false,
+                },
+            ],
+            slots: [
```
Comment (EisenbergEffect) [inline sites/site-utilities/src/definitions/fast-components/fast-accordion.definition.ts:27]: Does accordion not have a default slot?

Comment (eljefe223) [reply sites/site-utilities/src/definitions/fast-components/fast-accordion.definition.ts:27]: Nope only a slot for item

Comment (EisenbergEffect) [reply sites/site-utilities/src/definitions/fast-components/fast-accordion.definition.ts:27]: Do the accordion items automatically set their slot to item by any chance?

Comment (eljefe223) [reply sites/site-utilities/src/definitions/fast-components/fast-accordion.definition.ts:27]: yes they do

**Pull request merged**